### PR TITLE
Make meta.yaml valid YAML, re-render, run tests on Windows

### PR DESCRIPTION
**Pull request by inducer**

### CODE REVIEW

**Comment by inducer** [comment]
@conda-forge/core Any clues on what's causing this?
```
Traceback (most recent call last):
  File "/feedstock_root/ci_support/upload_or_check_non_existence.py", line 124, in <module>
    main()
  File "/feedstock_root/ci_support/upload_or_check_non_existence.py", line 104, in main
    exists = built_distribution_already_exists(cli, meta, owner)
  File "/feedstock_root/ci_support/upload_or_check_non_existence.py", line 24, in built_distribution_already_exists
    fname = bldpkg_path(meta)
TypeError: bldpkg_path() missing 1 required positional argument: 'config'
```

**Comment by jakirkham** [comment]
Please [re-render]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) this manually with `conda-smithy` version `1.3.2`.

**Comment by inducer** [comment]
Done, thanks for the tip.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -31,14 +31,14 @@ test:
...
         - pytest
     imports:
-        - pytools
-  commands:
-    - cd $SRC_DIR/test && python -m pytest  # [unix]
-  dev_url: http://github.com/inducer/pytools
+        - {{ name }}
+    commands:
+         - cd $SRC_DIR/test && python -m pytest  # [unix]
+    dev_url: http://github.com/inducer/{{ name }}
```
Comment (jakirkham) [inline recipe/meta.yaml]: The `dev_url` should be in the `about` section.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -31,14 +31,13 @@ test:
...
-  dev_url: http://github.com/inducer/pytools
+        - {{ name }}
+    commands:
+         - cd $SRC_DIR/test && python -m pytest  # [unix]
 
 
 about:
-    home: http://pypi.python.org/pypi/pytools
+    home: http://pypi.python.org/pypi/{{ name }}
     license: MIT
```
Comment (jakirkham) [inline recipe/meta.yaml]: Any chance there is a license file in the source that we can include in the package via `license_file`?

Comment (inducer) [inline recipe/meta.yaml]: Done.

**Comment by inducer** [comment]
Closed/reopened in an attempt get AppVeyor unstuck.

**Comment by inducer** [comment]
Travis stalled out: close/reopen to redo.

**Comment by pelson** [comment]
Windows failed with:

```
test_pytools.py ........F
================================== FAILURES ===================================
________________________________ test_diskdict ________________________________
    def test_diskdict():
        from pytools.diskdict import DiskDict
    
        from tempfile import NamedTemporaryFile
    
        with NamedTemporaryFile() as ntf:
>           d = DiskDict(ntf.name)
test_pytools.py:163: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
self = <pytools.diskdict.DiskDict object at 0x03D01E10>
dbfilename = 'c:\\users\\appveyor\\appdata\\local\\temp\\1\\tmp5skcqw'
version_base = (), dep_modules = [], commit_interval = 1
    def __init__(self, dbfilename, version_base=(), dep_modules=[],
            commit_interval=1):
>       self.db_conn = sqlite.connect(dbfilename, timeout=30)
E       OperationalError: unable to open database file
C:\conda\envs\_test\lib\site-packages\pytools\diskdict.py:50: OperationalError
===================== 1 failed, 18 passed in 3.47 seconds =====================
TESTS FAILED: pytools-2016.2.4-py27_0
```

**Comment by inducer** [comment]
AppVeyor failed with what looks like an unrelated issue. Open/Close to retry.

**Comment by jakirkham** [comment]
[Failure]( https://ci.appveyor.com/project/conda-forge/pytools-feedstock/build/1.0.16/job/8tix8olouiuh1rse#L196 ) appears related to [unicode characters]( https://github.com/conda-forge/pytools-feedstock/commit/a9457506b924bb1aa3a57bac87f1f0b402afdd9f#commitcomment-19371165 ). Please remove them and retry.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -31,22 +34,25 @@ test:
...
         - pytest
     imports:
-        - pytools
-  commands:
-    - cd $SRC_DIR/test && python -m pytest  # [unix]
-  dev_url: http://github.com/inducer/pytools
+        - {{ name }}
+    commands:
+         - cd $SRC_DIR/test && python -m pytest  # [unix]
+         - cd %SRC_DIR%/test && python -m pytest  # [win]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Corrected in following commit.

**Comment by jakirkham** [comment]
I think I pushed the needed fix to this PR. Let's see what the CIs do.

**Comment by inducer** [comment]
Thanks for pushing these fixes! (And sorry for messing those things up in the first place.)

**Comment by jakirkham** [comment]
No worries. Thanks for getting started on fixing it so quickly.

**Pull request merged**